README.md

```diff
@@ -33,9 +50,100 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for info on contributing to Cucumber.
 - Ruby 2.3
 - JRuby 9.2 (with [some limitations](https://github.com/cucumber/cucumber-ruby/blob/main/docs/jruby-limitations.md))
 
-## Code of Conduct
+### Ruby on Rails
+
+Using Ruby on Rails? You can use [cucumber-rails](https://github.com/cucumber/cucumber-rails)
+to bring Cucumber into your Rails project.
+
+## Usage
+
+### Initialization
+
+If you need to, initialize your `features` directory with
+
+    $ cucumber --init
+
+This will create the following directories and files if they do not exist already:
+
+    features
+    ├── step_definitions
+    └── support
+        └── env.rb
+
+### Creating your features and step definitions
+
+Create a file named `rule.feature` in the `features` directory with:
+
+```gherkin
+# features/rule.feature
+
+Feature: Rule Sample
 
-Everyone interacting in this codebase and issue tracker is expected to follow the Cucumber [code of conduct](https://cucumber.io/conduct).
+  Rule: This is a rule
+
+    Example: A passing example
+      Given this will pass
+      When I do an action
+      Then some results should be there
+
+    Example: A failing example
+      Given this will fail
+      When I do an action
+      Then some results should be there
+
+```
+
+And a file named `steps.rb` in `features/step_definitions` with:
+
+```ruby
+Given("this will pass") do
+  @this_will_pass = true
+end
+
+Given("this will fail") do
+  @this_will_pass = false
+end
+
+When("I do an action") do
+end
+
+Then("some results should be there") do
+  expect(@this_will_pass)
+end
+```
+
+### Running Cucumber
+
+    $ cucumber
+
+To execute a single feature file:
+
+    $ cucumber features/rule.feature
+
+To execute a single example, indicates the line of the name of the example:
+
+    $ cucumber features/rule.feature:7
+
+To summarize the results on the standard output, and writte a HTML report on disk:
+
+    $ cucumber --format summary --format html --out report.html
+
+For more command line options
+
+    $ cucumber --help
+
+You can also find documentation on the command line possibilities in
+[features/docs/cli](features/docs/cli).
+
+## To go further
+
+Where to get more info:
+
+- The main website: https://cucumber.io/
+- Documentation: https://cucumber.io/docs
+- Ruby API Documentation: http://www.rubydoc.info/github/cucumber/cucumber-ruby/
+- Support forum: https://groups.google.com/group/cukes
```

Comment:
https://community.smartbear.com/t5/Cucumber-Open/bd-p/CucumberOS is a newer forum, and we should promote that.

Reply:
Thank you

Yes, there is still room for improvement.
I have not reworked that last part indeed